core/client/app/components/gh-user-invited.js

```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```

Comment:
RE export, we used to always do `var Thing = {}; export default Thing;` to keep it consistent with the node code, but recently there was a discussion in which it was determined that `export default Ember.Thing.extend({})` makes more sense in ES6 code, so the Ember code is moving towards that instead.